smf/clickhouse/manifest.xml

```diff
@@ -4,7 +4,8 @@
 <service_bundle type='manifest' name='clickhouse'>
 
   <service name='system/illumos/clickhouse' type='service' version='1'>
```

Comment:
We should rename this to `oxide/clickhouse` probably.

Reply:
Done